Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Resources.EditorWindow
+{
+    public class GameLiftPlugin : UnityEditor.EditorWindow
+    {
+        [SerializeField] private Texture icon;
```

Comment:
question: how do you determine what is a `SerializeField` vs not?

Reply:
Understood. What icon is this? Just the icon for the overall plugin? Wondering i this can have more descriptive naming.

Reply:
Also could you help understand the naming convention for member variables.

Should this be _icon instead of icon?

Reply:
This is the icon that will be used in the editor window tab. 

Example:
![image](https://github.com/aws/amazon-gamelift-plugin-unity/assets/109524503/08a64b41-b29f-4bf3-84c7-e9de4b6506e1)

Reply:
I'm not 100% following. Usually there's an asset file (or scene file) of sorts where the icon would be captured when you provide it to a serialized field, but I'm not seeing anything like that in this PR nor am I seeing an icon displayed in the screenshots provided

Reply:
I didnt have the icon at the time of creating this PR, so I just created the variable for it to live in

Reply:
and it would be captured in the scripts .meta file